src/openproduct/urn/validators.py

```diff
@@ -0,0 +1,8 @@
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext_lazy as _
+
+
+def validate_urn_or_url(urn, url, field=""):
+    field += " " if field else ""
+    if not urn and not url:
+        raise ValidationError(_("een {}url of urn is verplicht").format(field))
```

Comment:
You can simplify it without the if
```suggestion
        raise ValidationError({field or "url_urn": [_('een url of urn is verplicht')]})
```